ee/api/test/test_funnel_correlation.py

```diff
@@ -1,9 +1,14 @@
-from typing import Any, Dict
+import dataclasses
```

Comment:
A file here instead: https://github.com/PostHog/posthog/blob/master/ee/clickhouse/views/test/test_clickhouse_insights.py#L59 seems like a good place for these tests